manifests/plugin/ansible.pp

```diff
@@ -31,6 +31,15 @@
     validate_absolute_path($working_dir)
   }
 
+  package { 'ansible':
```

Comment:
Likely to clash with other modules, consider using ensure_packages and possibly parameterising it.